doc/motion_planning_api/motion_planning_api_tutorial.rst

```diff
@@ -63,4 +58,4 @@ The entire code can be seen :codedir:`here in the moveit_tutorials GitHub projec
 
 The Launch File
 ---------------
-The entire launch file is :codedir:`here <motion_planning_api/launch/motion_planning_api_tutorial.launch>` on GitHub. All the code in this tutorial can be compiled and run from the moveit_tutorials package.
+The entire launch file is :codedir:`here <motion_planning_api/launch/motion_planning_api_tutorial.launch.py>` on GitHub. All the code in this tutorial can be compiled and run from the moveit_tutorials package.
```

Comment:
Link doesn't work for me. Perhaps it should be `:moveitdir:` instead (IIRC)

Reply:
I am not sure what to do about this.  It looks like `:codedir:` creates a link to the file on the main branch, but this file doesn't exist on the main branch yet.  I expect that the link will work once this PR is merged, but I'm not sure how to test it now.

Using `:moveitdir:` produced this warning:
`WARNING: Unknown interpreted text role "moveitdir".`

Reply:
OK, that's probably fine. We can keep an eye on it after merging and fix it with a new PR if needed.